java/code/src/com/suse/manager/webui/controllers/admin/AdminViewsController.java

```diff
@@ -171,19 +180,45 @@ public static ModelAndView showISSv3Peripherals(Request request, Response respon
      */
     private static ModelAndView showPeripheralDetails(Request request, Response response, User user) {
         long peripheralId = Long.parseLong(request.params("id"));
-        var peripheralData = Optional.ofNullable(HUB_FACTORY.findPeripheralById(peripheralId))
-            .map(PeripheralDetailsData::new)
-            .orElse(null);
-
+        PeripheralDetailsData peripheralData = Optional.ofNullable(HUB_FACTORY.findPeripheralById(peripheralId))
+                .map(PeripheralDetailsData::new)
+                .orElse(null);
         if (peripheralData == null) {
             throw Spark.halt(HttpStatus.SC_NOT_FOUND, "Peripheral not found");
         }
-
         Map<String, Object> data = new HashMap<>();
         data.put("peripheral", GSON.toJson(peripheralData));
         return new ModelAndView(data, "controllers/admin/templates/peripheral_details.jade");
     }
 
+    /**
+     * Show the details of an ISS v3 Peripheral
+     * @param request http request
+     * @param response http response
+     * @param user current user
+     * @return the view to show
+     */
+    private static ModelAndView showChannelSync(Request request, Response response, User user) {
+        long peripheralId = Long.parseLong(request.params("id"));
+        String peripheralFqdn;
+        ChannelSyncModel channelSyncModel;
+        try {
+            channelSyncModel = HUB_MANAGER.getChannelSyncModelForPeripheral(user, peripheralId);
+            peripheralFqdn = HUB_FACTORY.findPeripheralById(peripheralId).getFqdn();
+        }
+        catch (CertificateException eIn) {
+            throw Spark.halt(HttpStatus.SC_INTERNAL_SERVER_ERROR, LOC.getMessage("hub.invalid_root_ca"));
+        }
+        catch (IOException eIn) {
+            throw Spark.halt(HttpStatus.SC_INTERNAL_SERVER_ERROR, LOC.getMessage("hub.error_connecting_remote"));
```

Comment:
Spark.halt does not write a log message. I think it is better if we would also log an error. Otherwise a customer report and error, but we cannot find anything in the logs